src/cli/cli_tcp.cpp

```diff
@@ -425,6 +539,23 @@ template <> otError TcpExample::Process<Cmd("benchmark")>(Arg aArgs[])
             OutputLine("Untested");
         }
     }
+    /**
+     * @cli tcp benchmark run
+     * @code
+     * tcp benchmark run
+     * Done
+     * TCP Benchmark Complete: Transferred 73728 bytes in 7233 milliseconds
+     * TCP Goodput: 81.546 kb/s
+     * @endcode
+     * @cparam tcp benchmark run [@ca{size}]
+     * Use the `size` parameter to specify the number of bytes to send
+     * for the benchmark. If you do not use the `size` parameter, the default
+     * value is used.
```

Comment:
Can we note that the default value is `OPENTHREAD_CONFIG_CLI_TCP_DEFAULT_BENCHMARK_SIZE`